src/main/resources/create_tables.sql

```diff
@@ -0,0 +1,36 @@
+create type roles as enum ('jobseeker', 'recruiter')
+
+CREATE TABLE users (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    first_name VARCHAR(50) NOT NULL,
+    last_name VARCHAR(50) NOT NULL,
+    date_of_birth DATE NOT NULL,
+    gender VARCHAR(7) NOT NULL,
+    email VARCHAR(100),
+    status roles,
+    UNIQUE(email)
+);
+
+CREATE TABLE vacancies (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    vac_name VARCHAR(50),
```

Comment:
Can be simplified to
```suggestion
    name VARCHAR(50),
```

Reply:
corrected